style.css

```diff
@@ -5,189 +5,205 @@
 }
 /*CSS RESET*/
 
+@font-face {
+  font-family: "AVGARDD";
+  src: url(/fonts/AVGARDD.TTF);
```

Comment:
Aici ai incarcat fontul, dar ulterior trebuie sa aplici fontul la elementele unde il vrei aplicat. Uite aici un articol care iti arata cum sa faci asta https://www.digitalocean.com/community/tutorials/how-to-style-text-elements-with-font-size-and-color-in-css

Mai departe, cu privire la statement0urile care iti incarca un fisier extern, e bine ca atunci cand ai de specificat un path, sa pui acel path intre quotes: e.g. '/some/path' sau double quotes "/some/path". De ce? Pentru ca uneori poti avea spatii sau alte caractere care pot fi interpretate diferit si o nu o sa tio se incarce stilurile si o sa te intrebi de ce.